101-debugger/Tiltfile

```diff
@@ -0,0 +1,32 @@
+# -*- mode: Python -*
+
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-nodejs', port_forwards=[8000, 9229],
+    resource_deps=['deploy']
+)
```

Comment:
Nice! My first instinct is to document the Tiltfile a little more cuz our port forward syntax remains silly--but feel free to ignore if you don't think this is useful:
```
k8s_resource('example-nodejs', port_forwards=[
        8000,  # app itself
        9229   # debugger
    ], resource_deps=['deploy']
)
```

Reply:
i like that! done